src/Api/Response/Parser/FiltersParser.php

```diff
@@ -159,6 +159,14 @@ public function parseFilterItem($filterType, &$filterItem, array $data, $index)
         }
     }
 
+    private function escapeSpecialCharacters(string $string): string
+    {
+        $search = array('<', '"', "'");
```

Comment:
⚙️  To be more secure please add more characters to the filter that in the end this characters will filtered:

```
&    &amp;
<    &lt;
>    &gt;
"    &quot;
'    &#x27;
```

[Source](https://cheatsheetseries.owasp.org/cheatsheets/Cross_Site_Scripting_Prevention_Cheat_Sheet.html)

Reply:
> ⚙️ To be more secure please add more characters to the filter that in the end this characters will filtered:
> 
> ```
> &    &amp;
> <    &lt;
> >    &gt;
> "    &quot;
> '    &#x27;
> ```
> 
> [Source](https://cheatsheetseries.owasp.org/cheatsheets/Cross_Site_Scripting_Prevention_Cheat_Sheet.html)

Hello, I purposefully didn't add the rest because filters like categories have '>' and '&' in them - should I still add them?

Reply:
Thanks for your explanation. If the filter breaks the code, please leave it as it is.